go/chat/signencrypt/codec.go

```diff
@@ -157,10 +156,11 @@ func makeChunkNonce(nonce Nonce, chunkNum uint64) SecretboxNonce {
 	return &ret
 }
 
-func makeSignatureInput(plaintext []byte, encKey SecretboxKey, chunkNonce SecretboxNonce) []byte {
+func makeSignatureInput(plaintext []byte, encKey SecretboxKey, signaturePrefix libkb.SignaturePrefix, chunkNonce SecretboxNonce) []byte {
```

Comment:
I was expecting `panic`. I think any instance of `\x00` in the prefix is a dangerous programmer error. Agreed that adding an error return for this wouldn't make sense, either way.